src/main/resources/db/migration/V232__prefixer_standardbegrunnelser.sql

```diff
@@ -0,0 +1 @@
+--UPDATE vedtaksbegrunnelse SET vedtak_begrunnelse_spesifikasjon = concat('Standardbegrunnelse$', vedtak_begrunnelse_spesifikasjon);
```

Comment:
Forslag til migreringsscript for eksisterende verdier av standardbegrunnelser:
```suggestion
UPDATE vedtaksbegrunnelse SET vedtak_begrunnelse_spesifikasjon = concat('Standardbegrunnelse$', vedtak_begrunnelse_spesifikasjon);
```

Reply:
Ah nice!! Men - akkurat avslagsbegrunnelsene ligger som `vedtak_begrunnelse_spesifikasjoner` i tabell `VILKAR_RESULTAT`. Der vet vi at alle eksisterende verdier skal få prefix `Standardbegrunnelse$`

Men så må jeg sjekke ut de andre stedene der vi fremover lagrer ned `IVedtakBegrunnelse`. Jeg vet om tabellen `ENDRET_UTBETALING_ANDEL` men må sjekke om det er flere steder jeg har glemt å oppdatere

Det stedet du har funnet vil vi fremdeles kun lagre ned `Standardbegrunnelse` og det er typet som `Standardbegrunnelse` så jeg tror ikke vi skal gjøre akkurat den migreringen, for her bruker vi ikke `IVedtakBegrunnelseListConverter` når vi leser og skriver verdier til db

Reply:
Jeg kan tilpasse forslaget også spørre deg om hjelp igjen når jeg har kommet litt videre 😁